core/app/models/spree/promotion/rules/item_quantity.rb

```diff
@@ -0,0 +1,53 @@
+# A rule to apply to an order greater than (or greater than or equal to)
+# a specific quantity of line items
+module Spree
+  class Promotion
+    module Rules
+      class ItemQuantity < PromotionRule
+        preference :quantity_min, :integer, default: 1
+        preference :operator_min, :string, default: '>'
+        preference :quantity_max, :integer, default: 1000
+        preference :operator_max, :string, default: '<'
+
+
+        OPERATORS_MIN = ['gt', 'gte']
+        OPERATORS_MAX = ['lt','lte']
+
+        def applicable?(promotable)
+          promotable.is_a?(Spree::Order)
+        end
+
+        def eligible?(order, options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.